src/test/kotlin/org/jetBrains/research/tasktracker/tracking/DefaultContentProviderTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.jetBrains.research.tasktracker.tracking
+
+import org.jetBrains.research.tasktracker.config.MainTaskTrackerConfig.Companion.PLUGIN_NAME
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class DefaultContentProviderTest {
+
+    private lateinit var defaultContentProvider: DefaultContentProvider
+
+    @Before
+    fun setUp() {
+        defaultContentProvider = DefaultContentProvider
+    }
+
+    @Test
+    fun getDefaultFolderRelativePathTest() {
+        assertEquals("$PLUGIN_NAME/cpp", defaultContentProvider.getDefaultFolderRelativePath(task1))
+        assertEquals("$PLUGIN_NAME/kotlin", defaultContentProvider.getDefaultFolderRelativePath(task2))
+        assertEquals("$PLUGIN_NAME/java", defaultContentProvider.getDefaultFolderRelativePath(task3))
+        assertEquals("$PLUGIN_NAME/python", defaultContentProvider.getDefaultFolderRelativePath(task4))
+        assertEquals("$PLUGIN_NAME/jupyter", defaultContentProvider.getDefaultFolderRelativePath(task5))
+    }
```

Comment:
I'd prefer to avoid code duplicates, you can replace it with
```kotlin
    private fun Extension.getExpectedFolderName(): String = TODO("")

    @Test
    fun getDefaultFolderRelativePathTest() {
        listOf(task1, task2, task3, task4, task5).forEach { task ->
            assertEquals("Error message here", "$PLUGIN_NAME/${task.getExtension().getExpectedFolderName()}", defaultContentProvider.getDefaultFolderRelativePath(task))
        }
    }
```